core/services/workflows/models_yaml.go

```diff
@@ -21,10 +22,34 @@ func GenerateJsonSchema() ([]byte, error) {
 }
 
 func ParseWorkflowSpecYaml(data string) (workflowSpec, error) {
+	var url = "https://github.com/smartcontractkit/chainlink/"
 	w := workflowSpecYaml{}
 	err := yaml.Unmarshal([]byte(data), &w)
+	if err != nil {
+		return workflowSpec{}, err
+	}
+	schemaStr, err := GenerateJsonSchema()
```

Comment:
We generate the schema on every single workflow upload, kind of a waste of work, but it seems like a low volume operation to the point where it's extra complexity to cache

Reply:
I agree :) Probably not worth optimising at this stage.